src/main/resources/policies/zaak-rechten.rego

```diff
@@ -97,7 +97,7 @@ heropenen {
 
 default wijzigen_zaakdata := false
 wijzigen_zaakdata {
-    behandelaar.rol in user.rollen
+    beheerder.rol in user.rollen
```

Comment:
We have to change the policy table docu too

Reply:
Done :-)